pkg/dashboard/assets/js/cost_settings.js

```diff
@@ -25,6 +27,8 @@
   const costPerGBKey = "costPerGB";
   const otherOption = "Other";
   const emptyString = "";
+  const defaultCloudProvider = "AWS";
+  const defaultInstanceType = "663";
```

Comment:
This might be the issue--IDs are numbers, but this is a string. I think this should be `663`, not `"663"`

Reply:
Oh maybe not...I see you're using `option.value`. I wonder if it depends on the browser whether `option.value` gets converted to a string? I'm in firefox